jaeger-core/src/main/java/com/uber/jaeger/Span.java

```diff
@@ -223,6 +225,15 @@ public Span log(long timestampMicroseconds, Map<String, ?> fields) {
         if (logs == null) {
           this.logs = new ArrayList<LogData>();
         }
+
+        if (tracer.expandExceptionLogs()) {
+          Map<String, Object> errorFields = exceptionLogs(fields);
+          if (!errorFields.isEmpty()) {
+            errorFields.putAll(fields);
+            fields = errorFields;
```

Comment:
Tracer does not change explicitly logged items ( "error.object", "stack", "message" ..). It only derives new fields if are not logged.